Sources/Sentry/SentryTransaction.m

```diff
@@ -87,4 +83,49 @@ - (void)finish
     [self.hub captureTransaction:self];
 }
 
+- (SentrySpanId *)spanId
+{
+    return _spanContext.spanId;
+}
+
+- (SentryId *)traceId
+{
+    return _spanContext.traceId;
+}
+
+- (BOOL)isSampled
+{
+    return _spanContext.sampled;
+}
+
+- (NSString *)spanDescription
+{
+    return _spanContext.spanDescription;
+}
+
+- (void)setSpanDescription:(NSString *)spanDescription
+{
+    [_spanContext setSpanDescription:spanDescription];
+}
```

Comment:
fine using `spanDescription` as field name, but can we use just  `Description` for method and param names?

Reply:
those are the getters and setters for obj-c